test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,30 @@
 
 public class UtilsTest {
 
+   @Test
+   public void isAnyNull_null_returnsFalse() throws Exception {
+       assertHasNoNull();
+   }
+
+   @Test
+   public void isAnyNull_noNull_returnsFalse() throws Exception {
```

Comment:
Ideally, each test function should have more than one test. Otherwise, you can group multiple test cases together and not use the `scenarioBeingTested` and `expectedOutcome`.